executor/executor.cc

```diff
@@ -3,6 +3,8 @@
 
 // +build
 
+#include <sys/mman.h>
```

Comment:
Does it exist on windows?
If not, the windows build is probably already broken after 2c6621289f027, but let's not break it more.

Reply:
Ahh. I'll guard it with an #ifdef.